### ISSUE TITLE
Need to help

### ISSUE THREAD

**Issue by datking73** [prabhat-123/Smart-AI-Attendance-System-With-AntiSpoofing#2]
I'm Vietnam Student's .How can i contact you ?

**Comment by prabhat-123** [comment]
yeah you can cotact me on gmail : my gmail is srv.ale52@gmail.com................
What is your issue... Please mention.. I will try to solve your issue as soon as possible.
